src/Transports.Subscriptions.Abstractions/SubscriptionManager.cs

```diff
@@ -45,13 +46,18 @@ public SubscriptionManager(IGraphQLExecuter executer, ILoggerFactory loggerFacto
                 throw new ArgumentNullException(nameof(payload));
             if (context == null)
                 throw new ArgumentNullException(nameof(context));
+            if (_disposed)
+                throw new ObjectDisposedException(nameof(SubscriptionManager));
 
             var subscription = await ExecuteAsync(id, payload, context).ConfigureAwait(false);
 
             if (subscription == null)
                 return;
 
-            _subscriptions[id] = subscription;
+            if (_disposed)
```

Comment:
Concurrent dispose?

Reply:
The connection can disconnect during ExecuteAsync, so another check is needed

Reply:
OK. I still do not know this code.